<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signal_Monitor</class>
 <widget class="QMainWindow" name="Signal_Monitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTextEdit" name="ControlInfo">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Open New File --&amp;gt; Ctrl+O  |  Save File --&amp;gt; Ctrl+S  |  Exit Program --&amp;gt; Ctrl+X&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Pause Signal --&amp;gt; Spacebar |  Mov Signal Right --&amp;gt; Right Arrow | Mov Signal Left --&amp;gt; Left Arrow&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="Signal1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Signals</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="0">
          <layout class="QGridLayout" name="lyt_Signal">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="lyt_SignalDisplay">
             <item>
              <widget class="QGraphicsView" name="graphicsView"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="ControlLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <item row="1" column="1">
            <widget class="QPushButton" name="Signal1_Play_Stop">
             <property name="text">
              <string>Play</string>
             </property>
             <property name="shortcut">
              <string>Space</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="Signal1_Zoomout">
             <property name="text">
              <string>Zoom out</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="Signal1_PDF">
             <property name="text">
              <string>PDF</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="Signal1_Zoomin">
             <property name="text">
              <string>Zoom in</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="Signal1_MovRight">
             <property name="text">
              <string>Mov Right</string>
             </property>
             <property name="shortcut">
              <string>Right</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="Signal1_MovLeft">
             <property name="text">
              <string>Mov Left</string>
             </property>
             <property name="shortcut">
              <string>Left</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btn_addView">
        <property name="text">
         <string>Add View</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>654</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="ActionFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="ActionFile_OpenFile"/>
    <addaction name="ActionFile_SaveFile"/>
    <addaction name="separator"/>
    <addaction name="ActionFile_ExitFile"/>
   </widget>
   <widget class="QMenu" name="ActionSignal">
    <property name="title">
     <string>Signal</string>
    </property>
    <addaction name="ActionSignal_LinkSignals"/>
    <addaction name="ActionSignal_UnLinkSignals"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="ActionFile"/>
   <addaction name="ActionSignal"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ActionFile_OpenFile">
   <property name="text">
    <string>Open File             Ctrl+O</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="ActionFile_SaveFile">
   <property name="text">
    <string>Save File              Ctrl+S</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="ActionFile_ExitFile">
   <property name="text">
    <string>Exit                      Ctrl+X</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="ActionSignal_LinkSignals">
   <property name="text">
    <string>Link Signals</string>
   </property>
  </action>
  <action name="ActionSignal_UnLinkSignals">
   <property name="text">
    <string>UnLlink Signals</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
